app/src/lib/logging/logger.ts

```diff
@@ -64,12 +64,20 @@ function create(filename: string) {
 
 export function createLogger(directory: string): Promise<ILogger> {
   return new Promise<ILogger>((resolve, reject) => {
-    Fs.mkdir(directory, (error) => {
-      if (error) {
-        reject(error)
-      } else {
+    Fs.exists(directory, (exists) => {
```

Comment:
It's possible for us to run afoul of a time-to-check-time-to-use race. We should instead just try to create the directory and ignore failures that tell us it already exists.